go.mod

```diff
@@ -2,6 +2,8 @@ module github.com/nats-io/nats-server/v2
 
 go 1.22.0
 
+replace github.com/nats-io/jwt/v2 => github.com/nats-io/jwt/v2 v2.5.9-0.20240910093232-2e1251e51094
```

Comment:
Won't merge to main with this here. So need to do the JWT lib release first.

Reply:
Yep, as mentioned in the PR description.

Reply:
ok but to merge we need the jwt lib released and the go.mod properly updated etc..

Reply:
JWT library released & `go.mod` updated.